stable/kafka-pubsub-emulator/README.md

```diff
@@ -0,0 +1,87 @@
+# Pub/Sub Emulator for Kafka
+
+This chart packages the [Pub/Sub Emulator for Kafka](https://github.com/GoogleCloudPlatform/kafka-pubsub-emulator) which implements an emulation layer on top of an existing [Kafka](https://kafka.apache.org/) cluster.
+
+## Prerequisites
+
+* Kubernetes 1.4+ with Beta APIs enabled
+* A Kafka cluster
+
+## Chart Details
+
+This chart will create:
+
+1. A [StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) with a single replica running the Pub/Sub Emulator for Kafka
+
+2. A [Service](https://kubernetes.io/docs/concepts/services-networking/service/) exposing the Pub/Sub Emulator for Kafka
+
+3. A [ConfigMap](https://kubernetes.io/docs/tutorials/configuration/) that holds the Server configuration for the Pub/Sub Emulator for Kafka that will be mounted by the Pods in the StatefulSet above
+
+4. A [PersistentVolume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) that holds the projects layout (topics and subscriptions) configuration used by the Pub/Sub Emulator for Kafka. 
+
+### Installation
+
+```shell
+helm install --name emulator stable/kafka-pubsub-emulator<Paste>
```

Comment:
Did you mean to have `<Paste>` at the end of the line?

Reply:
I didn't, but apparently my editor thought otherwise... :)